extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -613,18 +625,6 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
 
-  /**
-   * @param {string} targetPageId
-   * @private
-   */
-  switchTo_(targetPageId) {
-    const payload = {targetPageId};
-    const eventInit = {bubbles: true};
-    dispatchCustom(this.win, this.element, EventType.SWITCH_PAGE, payload,
```

Comment:
Now that this event is gone, could you remove it from https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/0.1/events.js#L35 ?